runner/hooks/job-started.d/update-status

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env bash
 set -u
 
-exec update-status Running "Run $GITHUB_RUN_ID from $GITHUB_REPOSITORY" "$GITHUB_REPOSITORY" "$GITHUB_REPOSITORY_OWNER" "$GITHUB_WORKFLOW"
```

Comment:
I modified your work to not take those as function arguments, and instead refer to envvars presuming they are all optional, so that we won't bloat function parameters and arguments as we add more.